include/items.php

```diff
@@ -884,19 +909,35 @@ function item_store($arr,$force_parent = false, $notify = false, $dontcache = fa
 			logger('item_store: new item not found in DB, id ' . $current_post);
 	}
 
+	if ($arr['parent-uri'] === $arr['uri']) {
+		add_thread($current_post);
+	} else {
+		update_thread($parent_id);
+	}
+
+	q("COMMIT");
+
+	// Due to deadlock issues with the "term" table we are doing these steps after the commit.
+	// This is not perfect - but a workable solution until we found the reason for the problem.
 	create_tags_from_item($current_post);
 	create_files_from_item($current_post);
 
-	// Only check for notifications on start posts
-	if ($arr['parent-uri'] === $arr['uri']) {
-		add_thread($current_post);
-		q("COMMIT");
+	/**
```

Comment:
Please don't  use `/**` because this would initiate a doxygen block. Alternatives could be `/*` or `/* *`